packages/angular-workspace/tsconfig.json

```diff
@@ -36,7 +36,7 @@
       "es2020",
       "dom"
     ],
-    "importsNotUsedAsValues": "error",
+    "verbatimModuleSyntax": true,
     "ignoreDeprecations": "5.0",
```

Comment:
These two settings should be consistent across our tsconfigs. We could stage it in separate PRs essentially back-to-back or migrate everything together in a single PR.

Reply:
Updated `nimble-components`, `spright-compnents`, and `storybook` tsconfigs (and source files), too.